components/esp_http_server/src/esp_httpd_priv.h

```diff
@@ -471,6 +474,31 @@ int httpd_default_recv(httpd_handle_t hd, int sockfd, char *buf, size_t buf_len,
  * @}
  */
 
+/* ************** Group: WebSocket ************** */
+/** @name WebSocket
+ * Functions for WebSocket header parsing
+ * @{
+ */
+
+
+/**
+ * @brief   This function is for responding the client when in a WebSocket handshake phrase
+ *
+ * @param[in] hd    Server instance data
+ * @return
+ *  - ESP_OK                        : if sucessful
```

Comment:
Remove the "ifs" to be consistent with rest of the documentation?

Reply:
@shahpiyushv I've updated the docs here. Please check again if it's appropriate or not.